python/tk_desktop/desktop_window.py

```diff
@@ -715,27 +728,28 @@ def __launch_app_proxy_for_project(self, project, pipeline_configuration_id=None
             pipeline_configuration = None
             primary_pipeline_configuration = None
             for pc in pipeline_configurations:
+                # If we've stumbled upon the Primary.
                 if pc["code"] == constants.PRIMARY_PIPELINE_CONFIG_NAME:
                     primary_pipeline_configuration = pc
+                    # And there was no pipeline configuration saved from a last run.
```

Comment:
could the code be made easier to read if we named the last accessed pipeline config id something explicit rather than just putting that value into `pipeline_configuration_id` (line 722)?